### PR TITLE
en: Add doc enable_tls_for_ticdc.md

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added, or deleted? (Required)

Add the doc shows how to configure TiCDC to synchronizes data to a TLS-enabled downstream service

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1274/files
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/enable-tls-for-ticdc-sink.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Preparations
+
+Before you begin, do the following preparations:
+
+- Deploy a downstream service, and enable the TLS authentication on the client.
+- Generate the key file required for the client to access the downstream service.
+
+## Configure TiCDC to synchronizes data to a TLS-enabled downstream service
+
+1. Create a Kubernetes Secret object that contains a TLS certificate used to access the client of the downstream service. You can get the certificate from the key file you generated for the client.
```
Comment (KanShiori) [inline en/enable-tls-for-ticdc-sink.md:19]:
```suggestion
1. Create a Kubernetes Secret object that contains a client TLS certificate used to access the downstream service. You can get the certificate from the key file you generated for the client.
```

File: en/TOC.md [hunk 2]
```diff
@@ -38,6 +38,7 @@
   - [Enable TLS for the MySQL Client](enable-tls-for-mysql-client.md)
   - [Enable TLS between TiDB Components](enable-tls-between-components.md)
   - [Enable TLS for DM](enable-tls-for-dm.md)
+  - [Synchronize Data to TLS-enabled Downstream Services using TiCDC Components](enable-tls-for-ticdc-sink.md)
```
Comment (en-jin19) [inline en/TOC.md:41]:
```suggestion
  - [Replicate Data to TLS-enabled Downstream Services Using TiCDC Components](enable-tls-for-ticdc-sink.md)
```

File: en/enable-tls-for-ticdc-sink.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: Replicate Data to TLS-enabled Downstream Services using TiCDC Components
```
Comment (en-jin19) [inline en/enable-tls-for-ticdc-sink.md:2]:
```suggestion
title: Replicate Data to TLS-enabled Downstream Services Using TiCDC Components
```

File: en/enable-tls-for-ticdc-sink.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        baseImage: pingcap/ticdc
+        version: "v5.0.1"
+        # ...
+        tlsClientSecretNames:
+        - ${secret_name}
+    ```
+
+    Once the TiCDC Pod is running, the created Kubernetes Secret object is mounted to the TiCDC Pod. You can get the mounted key file in the `/var/lib/sink-tls/${secret_name}` directory of the Pod.
+
+3. Create a synchronization task using the `cdc cli` tool.
```
Comment (en-jin19) [inline en/enable-tls-for-ticdc-sink.md:51]:
```suggestion
3. Create a replication task using the `cdc cli` tool.
```

**Review by en-jin19 — COMMENTED** [review]
I have left some comments for consistency, please kindly take a look, thanks.

**Review by KanShiori — APPROVED** [review]
LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**